### PR TITLE
feat: [IDP-1336]: Updating descritpion for data points for data source screen

### PR DESCRIPTION
**Pull request by devesh-harness**
## Checklist
- [ ] I've documented the changes in the PR description.
- [ ] I've tested this change either in PR or local environment.
- [ ] If hashcheck failed I followed [the checklist](https://harness.atlassian.net/wiki/spaces/DEL/pages/21016838831/PR+Codebasehash+Check+merge+checklist) and updated this PR description with my answers to make sure I'm not introducing any breaking changes.

## Comment Triggers
<details>
  <summary>Build triggers</summary>

- Feature build: `trigger feature-build`
- 
  Specific builds
  - `trigger manager`
  - `trigger dms`
  - `trigger ng_manager`
  - `trigger cvng `
  - `trigger cmdlib`
  - `trigger template_svc`
  - `trigger events_fmwrk_monitor`
  - `trigger event_server`
  - `trigger change_data_capture`
  -  Trigger multiple builds together: For eg: `trigger dms, manager`
- Immutable delegate `trigger publish-delegate`
</details>

## [Latest PR Check Triggers](https://github.com/harness/harness-core/blob/develop/.github/pull_request_template.md)

<details>
  <summary>PR Check triggers</summary>
You can run multiple PR check triggers by comma separating them in a single comment. e.g. `trigger ut0, ut1`

- Compile: `trigger compile`
- CodeformatCheckstyle: `trigger checkstylecodeformat`
  - CodeFormat: `trigger codeformat`
  - Checkstyle: `trigger checkstyle`
- MessageMetadata: `trigger messagecheck`
- File-Permission-Check: `trigger checkpermission`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- Trigger CommonChecks: `trigger commonchecks`
- PMD: `trigger pmd`
- Copyright Check: `trigger copyrightcheck`
- Feature Name Check: `trigger featurenamecheck`
- UnitTests-ALL: `trigger uts`
- UnitTests-0: `trigger ut0`
- UnitTests-1: `trigger ut1`
- UnitTests-2: `trigger ut2`
- UnitTests-3: `trigger ut3`
- UnitTests-4: `trigger ut4`
- UnitTests-5: `trigger ut5`
- UnitTests-6: `trigger ut6`
- UnitTests-7: `trigger ut7`
- UnitTests-8: `trigger ut8`
- UnitTests-9: `trigger ut9`
- CodeBaseHash: `trigger codebasehash`
- CodeFormatCheckstyle: `trigger checkstylecodeformat`
- SonarScan: `trigger ss`
- GitLeaks: `trigger gitleaks`
- Trigger all Checks: `trigger smartchecks`
- Go Build: `trigger gobuild`
- Validate_Reviews: `trigger review`
- Module Dependency Check: `trigger mdc`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

## [Contributor license agreement](https://github.com/harness/harness-core/blob/develop/CONTRIBUTOR_LICENSE_AGREEMENT.md)

- - -
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/53818)

### CODE REVIEW

**Review by Debanitrkl — APPROVED** [review]

File: idp-service/src/main/resources/migrations/scorecard/dataPoints.json [hunk 1]
```diff
@@ -137,8 +137,8 @@
     "type": "BOOLEAN",
     "dataSourceIdentifier": "pagerduty",
     "dataSourceLocationIdentifier": "pagerduty_service_directory",
-    "description": "Short description for the data point",
-    "detailedDescription": "Short Description for data point here \n\n**How it's calculated:** Idea about the calculation of the data point\n\n**Prerequisites:** Prerequisites if any for calculation of data point"
+    "description": "Checks if escalation policy is set for given service.",
+    "detailedDescription": "**How it's calculated:** For the given service on PagerDuty it checks if on escalation policy is set or not.\n\n**Prerequisites:** The Pagerduty plugin needs to be configured and enabled in the admin section. Refer: [Link](https://developer.harness.io/docs/internal-developer-portal/plugins/available-plugins/pagerduty/)"
```
Comment (Debanitrkl) [inline idp-service/src/main/resources/migrations/scorecard/dataPoints.json:141]:
```suggestion
    "detailedDescription": "**How it's calculated:** For the given service on PagerDuty it checks if any escalation policy is set or not.\n\n**Prerequisites:** The Pagerduty plugin needs to be configured and enabled in the admin section. Refer: [Link](https://developer.harness.io/docs/internal-developer-portal/plugins/available-plugins/pagerduty/)"
```

**Review by vikyathharekal — DISMISSED** [review]

**Comment by devesh-harness** [comment]
trugger uts

**Comment by devesh-harness** [comment]
trigger uts

**Comment by devesh-harness** [comment]
trigger messagecheck

**Comment by devesh-harness** [comment]
trigger uts

**Comment by devesh-harness** [comment]
trigger uts

**Review by vikyathharekal — APPROVED** [review]

**Review by Aashrit-Harness — APPROVED** [review]

**Comment by devesh-harness** [comment]
trigger review

**Pull request merged**